pallets/funding/src/lib.rs

```diff
@@ -555,17 +542,39 @@ pub mod pallet {
 		StorageMap<_, Blake2_128Concat, Did, BoundedVec<ProjectId, MaxParticipationsForMaxMultiplier>, ValueQuery>;
 
 	#[pallet::storage]
+	pub type UserMigrations<T: Config> = StorageDoubleMap<
+		_,
+		Blake2_128Concat,
+		ProjectId,
+		Blake2_128Concat,
+		T::AccountId,
+		(MigrationStatus, BoundedVec<Migration, MaxParticipationsPerUser<T>>),
+	>;
+
+	pub struct MaxParticipationsPerUser<T: Config>(PhantomData<T>);
```

Comment:
Can't make it in parameter_types from T:: as we cannot have generic variables in parameter_types (at least I don't think so after trying). We could replace this with an extra Config item and then have a sanity check that it would equal to the three different items summed. However, I'm not sure if I prefer that solution. Wdyt?